deployments/clowder.yaml

```diff
@@ -235,7 +233,5 @@ parameters:
     value: "true"
     description: |
       It allows to enable / disable RBAC middleware.
-  - name: CLIENTS_RBAC_BASE_URL
-    required: true
-    description: |
-      Point out to the rbac service base url
+  - name: SSL_CERT_DIR
```

Comment:
Could you please add a comment explaining where the TLS CA certs are coming from (clowder operator populates the cdapp directory with trust roots).

Reply:
:+1: thanks mate! updated!